### PR TITLE
Switch between 2 and 3 parameter cost coverage curves

### PR DESCRIPTION
**Pull request by robynstuart**
https://trello.com/c/lz2WvNRs/444-add-additional-options-for-non-hiv-and-daly-benefits-to-cost-coverage-outcome-page
https://trello.com/c/nbq3lHt1/287-add-additional-options-for-non-hiv-and-daly-benefits

### CODE REVIEW

**Comment by Ashalynd** [comment]
Either I am going to look at this one, or whoever merged this, do not do it without going through the whole UI flow :-}

File: server/src/sim/makeccocs.py [hunk 1]
```diff
@@ -19,11 +19,12 @@
 
 ## Set defaults for testing makeccocs
 default_progname = 'SBCC'
-default_ccparams = [] # [0.9, 0.28, 154000.0, 1.0]
-default_ccplot = [] # [2e6, [0, []]]
-default_coparams = [] # [0.3, 0.5, 0.7, 0.9]
```
Comment (Ashalynd) [inline server/src/sim/makeccocs.py]: @robynstuart let's not assume 'nan' in anything that potentially comes from frontend.

Comment (robynstuart) [inline server/src/sim/makeccocs.py]: @Ashalynd what will come from the frontend if a textbox is left blank?

Comment (Ashalynd) [inline server/src/sim/makeccocs.py]: None. Can it be left blank? So only first 3 fields are required?

Comment (robynstuart) [inline server/src/sim/makeccocs.py]: yes, that's the plan.

Comment (Ashalynd) [inline server/src/sim/makeccocs.py]: OK, replacing the default values with None should not do much harm as I see.

Comment (robynstuart) [inline server/src/sim/makeccocs.py]: No that should be fine. I didn't know how they would come from the frontend but I assumed at least they wouldn't be floats, so the rest of the logic is based on that. I can simplify it later.

**Pull request merged**